components/molecule/rating/package.json

```diff
@@ -9,7 +9,8 @@
     "build:styles": "../../../node_modules/.bin/cpx './src/**/*.scss' ./lib"
   },
   "dependencies": {
-    "@s-ui/component-dependencies": "1"
+    "@s-ui/component-dependencies": "1",
+    "@s-ui/react-atom-icon": "^1.6.0"
```

Comment:
fix the version:
`"@s-ui/react-atom-icon": "1"`

Reply:
Fixed!